libs/database/src/lib/entities/i18n-item.entity.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model,
+  PrimaryKey,
+  Table,
+} from "sequelize-typescript";
+import { BIGINT, STRING } from "sequelize";
+
+@Table({ tableName: "i18n_items", underscored: true, paranoid: false })
+export class i18nItem extends Model<i18nItem> {
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(BIGINT.UNSIGNED)
+  override id: number;
+
+  @AllowNull
+  @Column(STRING)
+  status: string | null;
+
+  @AllowNull
+  @Column(STRING)
+  type: string | null;
+
+  @AllowNull
+  @Column(STRING)
+  shortValue: string | null;
+
+  @AllowNull
+  @Column(STRING)
```

Comment:
Looking at the current schema, this should be a `TEXT`, not a `STRING`. We need to keep these types as close to what's in the current schema as possible because this codebase will eventually become the source of truth for what the schema in the prod DB will be.